app-modules/contact/src/Filament/Resources/OrganizationResource.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Aiding App™ is licensed under the Elastic License 2.0. For more details,
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AidingApp\Contact\Filament\Resources;
+
+use Filament\Resources\Resource;
+use Illuminate\Database\Eloquent\Builder;
+use AidingApp\Contact\Models\Organization;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+use AidingApp\Contact\Filament\Resources\OrganizationResource\Pages\EditOrganization;
+use AidingApp\Contact\Filament\Resources\OrganizationResource\Pages\ViewOrganization;
+use AidingApp\Contact\Filament\Resources\OrganizationResource\Pages\ListOrganizations;
+use AidingApp\Contact\Filament\Resources\OrganizationResource\Pages\CreateOrganization;
+
+class OrganizationResource extends Resource
+{
+    protected static ?string $model = Organization::class;
+
+    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';
+
+    protected static ?int $navigationSort = 21;
+
+    public static function getPages(): array
+    {
+        return [
+            'index' => ListOrganizations::route('/'),
+            'create' => CreateOrganization::route('/create'),
+            'view' => ViewOrganization::route('/{record}'),
+            'edit' => EditOrganization::route('/{record}/edit'),
+        ];
+    }
+
+    public static function getEloquentQuery(): Builder
+    {
+        return parent::getEloquentQuery()
+            ->withoutGlobalScopes([
+                SoftDeletingScope::class,
+            ]);
+    }
```

Comment:
@danharrin and @ankit-canyon I forget, does this result in your seeing soft deleted modals on pages?

Reply:
@Orrison [It looks like](https://filamentphp.com/docs/3.x/panels/resources/deleting-records#adding-soft-deletes-to-an-existing-resource) we also need to add the `TrashedFilter` and it should fix it.

Reply:
@Orrison 
As we discussed in call, I removed this code.